serverless/functions/how-to/add-trigger-to-a-function.mdx

```diff
@@ -41,6 +41,8 @@ For each message that is sent to an SQS queue, the SQS trigger reads the message
 The function can then process the message and perform any required actions, such as updating a database or sending a notification.
 Currently, SQS queues of the [Scaleway Messaging and Queuing](https://www.scaleway.com/en/docs/serverless/messaging/quickstart/) product are supported.
 
+The configuration of the queue retention may affect the behavior of the trigger. See [How to configure event retention for trigger inputs](/serverless/functions/reference-content/configure-trigger-inputs/).
```

Comment:
Refer to the [How to configure event retention for trigger inputs](/serverless/functions/reference-content/configure-trigger-inputs/) page for more information.